docs/index.html

```diff
@@ -2,6 +2,23 @@
 <html lang="en">
   <head>
   	<meta charset="utf-8">
+	  
```

Comment:
Wrap this list element in a <nav> tag.  https://developer.mozilla.org/en-US/docs/Web/HTML/Element/nav  make sure to give it a title/role/aria-label attribute describing the landmark element.

Reply:
This is also an invalid location for this list/nav element. The only elements that should be located inside the <head> tag are <title>,<link><meta><script> or <style> tags or similar tags that describe the html. 
Semantic/Landmark elements all need to be contained within the <body></body> tag.  

Solution: Move right below opening <body> tag.

Reply:
I believe I have this. Please let me know if its still off.